pkg/kubelet/stats/helper.go

```diff
@@ -127,17 +127,30 @@ func cadvisorInfoToNetworkStats(name string, info *cadvisorapiv2.ContainerInfo)
 	if !found {
 		return nil
 	}
+
+	iStats := statsapi.NetworkStats{}
+	isDefaultInterfaceFound := false
 	for _, inter := range cstat.Network.Interfaces {
+		iStat := statsapi.InterfaceStats{
+			Time:     metav1.NewTime(cstat.Timestamp),
+			RxBytes:  &inter.RxBytes,
+			RxErrors: &inter.RxErrors,
+			TxBytes:  &inter.TxBytes,
+			TxErrors: &inter.TxErrors,
+		}
+
 		if inter.Name == network.DefaultInterfaceName {
-			return &statsapi.NetworkStats{
-				Time:     metav1.NewTime(cstat.Timestamp),
-				RxBytes:  &inter.RxBytes,
-				RxErrors: &inter.RxErrors,
-				TxBytes:  &inter.TxBytes,
-				TxErrors: &inter.TxErrors,
-			}
+			isDefaultInterfaceFound = true
+			iStats.InterfaceStats = iStat
 		}
+
+		iStats.Interfaces = append(iStats.Interfaces, iStat)
+	}
+
+	if isDefaultInterfaceFound {
+		return &iStats
 	}
```

Comment:
I think you can remove the check for the default interface now.

Reply:
Will do.